micro-slack/src/main/java/com/aol/micro/server/slack/SlackConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package com.aol.micro.server.slack;
+
+import lombok.Getter;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+@Getter
+public class SlackConfiguration {
+
+    private final String webhookUri;
+
+    @Autowired
+    public SlackConfiguration(@Value("${slack.webhookUri}") String webhookUri){
```

Comment:
I prefer the config parameter name prefixed with `micro`, for eg `micro.slack.webhook.uri`